docs/usage.md

```diff
@@ -91,3 +93,143 @@ Find and set the options which fit to your demands, it is also an option to chan
+1. Attach the boot media to your system
+1. Start up the system the first time
+1. Wait for completion of the automated setup
+
+### Options within the file
+
+The options are generally described within the file itself. Basically see the file contents for details about the configuration options.  
+Below, only the most important options are listed. For further options go into the file `dietpi.txt` itself.
+
+#### Network configuration
+
+To achieve the basic network configuration the following options shall be taken into account:
+
+- Run with WiFi: AUTO_SETUP_NET_WIFI_ENABLED, AUTO_SETUP_NET_WIFI_COUNTRY_CODE in combination with `dietpi-wifi.txt`
+- Use a static IP address: AUTO_SETUP_NET_USESTATIC, AUTO_SETUP_NET_STATIC_IP, AUTO_SETUP_NET_STATIC_MASK, AUTO_SETUP_NET_STATIC_GATEWAY, AUTO_SETUP_NET_STATIC_DNS, AUTO_SETUP_DHCP_TO_STATIC
+- Usage of a proxy server: CONFIG_PROXY_ADDRESS, CONFIG_PROXY_PORT, CONFIG_PROXY_USERNAME, CONFIG_PROXY_PASSWORD
+
+#### System options
+
+Many of the system options can be set with the automated installation and can be also changed afterwards via `dietpi-config`:
+
+- Basic settings: AUTO_SETUP_NET_HOSTNAME, AUTO_SETUP_GLOBAL_PASSWORD
+- Localization: AUTO_SETUP_LOCALE, AUTO_SETUP_KEYBOARD_LAYOUT, AUTO_SETUP_TIMEZONE
+- Autostart: AUTO_SETUP_AUTOSTART_TARGET_INDEX, AUTO_SETUP_AUTOSTART_LOGIN_USER
+- Overclocking: CONFIG_CPU_GOVERNOR, CONFIG_CPU_MAX_FREQ, CONFIG_CPU_MIN_FREQ, etc.
+- Auto-updating: CONFIG_CHECK_DIETPI_UPDATES, CONFIG_CHECK_APT_UPDATES
+
+#### Software preferences
+
+Some server options can be set via AUTO_SETUP_SSH_SERVER_INDEX, AUTO_SETUP_FILE_SERVER_INDEX, AUTO_SETUP_WEB_SERVER_INDEX
+
+#### Software packages installation
+
+All software packages, which can be installed via `dietpi-software` can also be installed automatically using the keyword AUTO_SETUP_INSTALL_SOFTWARE_ID.  
+The software packages are given with their software ID (a number) which can be found on the left side when browsing the software in the `dietpi-software` script. Additionally they are given in the [list of available software IDs](https://github.com/MichaIng/DietPi/wiki/DietPi-Software-list).
+
+As an example the software package "Plex Media Server" has the ID 42.
+
+#### User script
+
+Execution of a user script can be done at these two instants of time:
+
+- pre-networking and pre-DietPi install (`/boot/Automation_Custom_PreScript.sh`)
+- post-networking and post-DietPi install (`/boot/Automation_Custom_Script.sh`)
+
+Search `AUTO_SETUP_CUSTOM_SCRIPT_EXEC` for details.
+
+#### System restore
+
+A restore process can be executed automatically using the keyword AUTO_SETUP_BACKUP_RESTORE. This gives the option to start up a system with a previous made system backup (e.g. for generating many identical systems).
+
+#### General options
+
+These keywords are general ones and need to be used for fully automated setups:
+
+- AUTO_SETUP_ACCEPT_LICENSE
+- AUTO_SETUP_AUTOMATED
```

Comment:
If `AUTO_SETUP_AUTOMATED=1` is set, then `AUTO_SETUP_ACCEPT_LICENSE=1` is implicitly set, so the latter then does not need to be changed: https://github.com/MichaIng/DietPi/blob/733d81e/dietpi.txt#L121

Reply:
Added a remark.  
In the examples I took that into account already.